packages/x-data-grid-premium/src/hooks/features/aggregation/gridAggregationInterfaces.ts

```diff
@@ -74,7 +74,7 @@ export interface GridAggregationFunction<V = any, AV = V> {
   getCellValue?: (params: GridAggregationGetCellValueParams) => V;
 }
 
-interface GridAggregationParams<V = any> {
+export interface GridAggregationParams<V = any> {
```

Comment:
May I ask why is this export needed along with this PR, or do we need to rebase with the master branch?

Reply:
Not sure why that's in there as I didn't write that line

Reply:
Could you try merging the latest `master` branch?

Reply:
Confirming that `upstream/master` should be merged into this branch. 😉

Reply:
I just did that earlier, am I missing something that I'm doing wrong?

Reply:
Are you sure that you merged `upstream/master` and not your out of date fork's master? 🤔

Reply:
You're right, I missed that, should be updated now

Reply:
It's all good, I can see where the misunderstanding came from. 
I followed the CONTRIBUTING.md which linked to the CONTRIBUTING.md from material-ui [https://github.com/mui/material-ui/blob/HEAD/CONTRIBUTING.md](url)
Which talks about using the next branch.

I'm not too familiar with rebasing but I'll try it, if that doesn't work out I'll open a new PR from the correct branch

Reply:
@magnimarels I have rebased the branch onto the latest `upstream/master` and I've discarded the oldest commits which looked like duplicates. 😉 
Please double-check if everything looks correct. 🙏 
I'm leaving the final review to @mui/xgrid. 👍